new.html

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
I noticed the Doctype, HTML and Viewport are missing from your HTML document, I think it will be good if you can  include them, as the Doctype specifically informs web browser about the type and version of HTML used in building the web document and Viewport helps mitigate the problem of the virtual viewport(user's view area) on narrow screen devices